packages/selected-network-controller/CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.0.0]
```

Comment:
I bumped the major version on this since the peer dependency on `@metamask/network-controller` is also being bumped in this way.